packages/labextension/src/index.ts

```diff
@@ -2,17 +2,26 @@ import {
   JupyterFrontEnd,
   JupyterFrontEndPlugin,
 } from '@jupyterlab/application';
+import { INotebookTracker } from '@jupyterlab/notebook';
+import { LabIcon } from '@jupyterlab/ui-components';
+import { ICommandPalette } from '@jupyterlab/apputils';
+import { KernelUsagePanel } from './panel';
+import tachometer from '../style/tachometer.svg';
 
 import { IStatusBar } from '@jupyterlab/statusbar';
 
 import { ITranslator } from '@jupyterlab/translation';
 
 import { MemoryUsage } from './memoryUsage';
 
+namespace CommandIDs {
+  export const getKernelUsage = 'kernel-usage:get';
+}
+
 /**
  * Initialization data for the jupyter-resource-usage extension.
  */
-const extension: JupyterFrontEndPlugin<void> = {
+const memory_extension: JupyterFrontEndPlugin<void> = {
```

Comment:
Minor code style nitpick:

```suggestion
const memoryStatusPlugin: JupyterFrontEndPlugin<void> = {
```